appserver/ejb/ejb-http-remoting/admin/src/main/java/fish/payara/ejb/http/admin/EjbInvokerService.java

```diff
@@ -129,9 +129,14 @@ public void event(EventListener.Event event) {
     public UnprocessedChangeEvents changed(PropertyChangeEvent[] events) {
         List<UnprocessedChangeEvent> unchangedList = new ArrayList<>();
         for (PropertyChangeEvent event : events) {
+            if ("enabled".equals(event.getPropertyName())){
```

Comment:
This is **still** too broad :P

For example, if the service isn't enabled and you do `set-ejb-invoker-configuration --securityenabled true` it'll say a restart is required, which I believe is very unlikely to be true.

You probably need to add a check for if the service is enabled.